arwen/elrondapi/elrondei.go

```diff
@@ -891,8 +888,7 @@ func executeOnSameContext(
 	gasToUse += metering.GasSchedule().BaseOperationCost.DataCopyPerByte * uint64(actualLen)
 	metering.UseGas(gasToUse)
 
-	invBytes := arwen.InverseBytes(value)
-	bigIntVal := big.NewInt(0).SetBytes(invBytes)
+	bigIntVal := big.NewInt(0).SetBytes(value)
```

Comment:
I am not sure that these api's would work without inversion.

Reply:
Added tests specifically for `getCallValue()` and `int64getArguments()`. Tests for `executeOn(Same|Dest)Context()` are on the execution tests branch.